collvoid:
  controllers:
#    - geofencing_node
#    - simple_collvoid
#    - nn_controller_dwm1000_discrete_particle_multi
    - nn_controller_dwm1000_discrete_particle_multi_simple
  geofencing:
    min_x: -1.60
    min_y: -2.1
    max_x: 2.3
    max_y: 1.5
    recover_speed: 0.3
    buffer_len: 0.05
  simple_collvoid:
    min_xy_dist: 0.17
    max_xy_dist: 0.30
    max_xy_avoid_vel: 0.4
    max_xy_goal_vel: 0.3
    error_factor: 1.7
  nn_controller_dwm1000:
    dwm1000_active: False
#    dwm1000_active: True
    dwm_timeout: 1.0
    avg_buffer_size: 1
#    movement_config_file: "/home/broecker/src/cf_nn_weights/v2/movement/drqn_3/nn_config_drqn.yaml"
    movement_config_file: "/home/broecker/src/cf_nn_weights/v2/movement/simple_dqn/dqn_2/nn_config.yaml"
    angle_config_file: "/home/broecker/src/cf_nn_weights/v2/angle_discrete/orientation_joint/rnn_1/nn_config.yaml"
    max_speed_limit: 0.45
    min_speed_limit: 0.15
    speed_limit_max_distance: 1.5
    speed_limit_min_distance: 0.45
    update_rate: 8.0

    goal_samples_size: 30
    spawn_new_goal_samples: 0.1
    goal_resample_x_limit: 0.05
    goal_resample_y_limit: 0.05
    goal_resample_yaw_limit: 0.1
    goal_respawn_yaw_limit: 0.1
    goal_respawn_y_limit: 0.01
    goal_respawn_x_limit: 0.01
    goal_resample_prob: 0.95

    obs_samples_size: 70
    spawn_new_obs_samples: 0.1
    #in meter
    obs_resample_x_limit: 0.3
    obs_resample_y_limit: 0.3
    #percent of pi
    obs_resample_yaw_limit: 0.1
    obs_respawn_yaw_limit: 0.1
    #in meter
    obs_respawn_y_limit: 0.1
    obs_respawn_x_limit: 0.1
    obs_resample_prob: 0.70



